variables.tf

```diff
@@ -16,10 +16,11 @@ variable "bucket_versioning" {
 
 variable "tags" {
   description = "A map of tags (key-value pairs) passed to resources."
-  type        = "map"
+  type        = map(string)
   default     = {}
 }
 
 variable "site_name" {
   description = "The name of the certificate and address for the site"
```

Comment:
I would also suggest adding the type signature for all your variables (informative error messages), and using `bool` instead of `string` for e.g. `bucket_versioning` - it should work now 😅

Reply:
Yes, good idea!